sdk/mongocluster/arm-mongocluster/review/arm-mongocluster.api.md

```diff
@@ -0,0 +1,478 @@
+export interface PrivateEndpointConnectionsListByMongoClusterOptionalParams extends OperationOptions {
+}
+
+// @public (undocumented)
+export interface PrivateEndpointConnectionsOperations {
+    // (undocumented)
+    create: (subscriptionId: string, resourceGroupName: string, mongoClusterName: string, privateEndpointConnectionName: string, resource: PrivateEndpointConnectionResource, options?: PrivateEndpointConnectionsCreateOptionalParams) => PollerLike<OperationState<void>, void>;
+    // (undocumented)
+    delete: (subscriptionId: string, resourceGroupName: string, mongoClusterName: string, privateEndpointConnectionName: string, options?: PrivateEndpointConnectionsDeleteOptionalParams) => PollerLike<OperationState<void>, void>;
+    // (undocumented)
+    get: (subscriptionId: string, resourceGroupName: string, mongoClusterName: string, privateEndpointConnectionName: string, options?: PrivateEndpointConnectionsGetOptionalParams) => Promise<PrivateEndpointConnectionResource>;
+    // (undocumented)
+    listByMongoCluster: (subscriptionId: string, resourceGroupName: string, mongoClusterName: string, options?: PrivateEndpointConnectionsListByMongoClusterOptionalParams) => PagedAsyncIterableIterator<PrivateEndpointConnectionResource>;
+}
+
+// @public
+export type PrivateEndpointServiceConnectionStatus = string;
+
+// @public
+export interface PrivateLinkResource extends ProxyResource {
+    properties?: PrivateLinkResourceProperties;
+}
+
+// @public
+export interface PrivateLinkResourceListResult {
+    nextLink?: string;
+    value: PrivateLinkResource[];
+}
+
+// @public
+export interface PrivateLinkResourceProperties {
+    readonly groupId?: string;
+    readonly requiredMembers?: string[];
+    requiredZoneNames?: string[];
+}
+
+// @public
+export interface PrivateLinkServiceConnectionState {
+    actionsRequired?: string;
+    description?: string;
+    status?: PrivateEndpointServiceConnectionStatus;
+}
+
+// @public (undocumented)
+export interface PrivateLinksListByMongoClusterOptionalParams extends OperationOptions {
+}
+
+// @public (undocumented)
+export interface PrivateLinksOperations {
+    // (undocumented)
+    listByMongoCluster: (subscriptionId: string, resourceGroupName: string, mongoClusterName: string, options?: PrivateLinksListByMongoClusterOptionalParams) => PagedAsyncIterableIterator<PrivateLinkResource>;
+}
+
+// @public
+export type ProvisioningState = string | ResourceProvisioningState | "InProgress" | "Updating" | "Dropping";
+
+// @public
+export interface ProxyResource extends Resource {
+}
+
+// @public
+export type PublicNetworkAccess = string;
+
+// @public
+export interface Resource {
+    readonly id?: string;
+    readonly name?: string;
+    readonly systemData?: SystemData;
+    readonly type?: string;
+}
+
+// @public
+export type ResourceProvisioningState = string;
+
+// Warning: (ae-forgotten-export) The symbol "DocumentDBContext" needs to be exported by the entry point index.d.ts
```

Comment:
This doesn't look like updated with the latest codegen?

Reply:
update code with latest codegen and all the test cases are passed